CONTRIBUTING.md

```diff
@@ -1,17 +1,16 @@
-Contributing
-============
+# Contributing
```

Comment:
Prettier auto-formatted the markdown again. 😞

Reply:
perhaps we should have a linter for markdown, so we have a consistent markdown formatter and we would avoid this kind of change

Reply:
I think my vscode is using `prettier`, which we also use for our UI code, so that might be the easiest answer. Out of curiosity, what text editor are you using? It'd be nice to pick something that integrates easily for everyone. I'll create an issue with some notes, so we can dig into this in a follow-up.

Reply:
I'm using IntelliJ for everything

Reply:
👍  created https://github.com/perses/perses/issues/889 to track looking into this for when one of us has time.